app/views/notifications/_emergency_banner.html.erb

```diff
@@ -1,9 +1,9 @@
-<section id="emergency-banner-notification" class="<%= banner.campaign_class %>">
+<section id="emergency-banner-notification" class="<%= banner.campaign_class %>" role="banner">
   <div>
-    <p><%= banner.heading %><br>
-    <%= banner.short_description %></p>
+    <h1><%= banner.heading %></h1>
```

Comment:
The outline is still in the HTML5 spec but it's actually not interpreted by any browser. That's why there is a warning in the spec to say so: http://w3c.github.io/html/sections.html#creating-an-outline